repose-aggregator/core/repose-core/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     compile "io.opentracing:opentracing-util"
     compile "com.uber.jaeger:jaeger-core"
 
+    compileOnly "javax.inject:javax.inject"
```

Comment:
It looks like there currently a `compile` dependency on this that I believe should be removed.